movemeant/push_notifications/models.py

```diff
@@ -10,6 +10,7 @@
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django.utils.encoding import smart_str
+from pyfcm import FCMNotification
```

Comment:
is this dependency added to the requirements file?

Reply:
Just added the dependency and created a fresh pull request